common/app/conf/audio/Flagship.scala

```diff
@@ -0,0 +1,10 @@
+package conf.audio
+
+object Flagship {
```

Comment:
this file was deleted in an earlier commit (replaced with FlagshipFrontContainer.scala)